packages/admin-next/dashboard/src/hooks/api/invites.tsx

```diff
@@ -5,29 +5,34 @@ import {
   useMutation,
   useQuery,
 } from "@tanstack/react-query"
-import { client } from "../../lib/client"
+import { sdk } from "../../lib/client"
 import { queryClient } from "../../lib/medusa"
 import { queryKeysFactory } from "../../lib/query-key-factory"
-import { CreateInviteReq } from "../../types/api-payloads"
 import {
-  InviteDeleteRes,
-  InviteListRes,
-  InviteRes,
-} from "../../types/api-responses"
+  AdminInviteResponse,
+  DeleteResponse,
+  HttpTypes,
+  PaginatedResponse,
+} from "@medusajs/types"
 
 const INVITES_QUERY_KEY = "invites" as const
 const invitesQueryKeys = queryKeysFactory(INVITES_QUERY_KEY)
 
 export const useInvite = (
   id: string,
   options?: Omit<
-    UseQueryOptions<InviteRes, Error, InviteRes, QueryKey>,
+    UseQueryOptions<
+      { invite: HttpTypes.AdminInviteResponse },
+      Error,
+      { invite: HttpTypes.AdminInviteResponse },
```

Comment:
Q: why do we have the res type twice in many places? It might be a dumb question 😅